components/ChatMessage.vue

```diff
@@ -76,8 +76,29 @@
     </div>
     <chat-message-warning v-if="phoneNumber" />
     <chat-message-date-read :id="id" :chatid="chatid" :last="last" :pov="pov" />
+
+    <GeneralSuccessModal
+      ref="deleteMessageSuccessModal"
+      message="Successfully deleted the message!"
+    />
+    <ForgetFailModal ref="deleteMessageFailModal" />
```

Comment:
I don't think we should be using `ForgetFailModal` as the error modal here.  You should introduce something like `GeneralSuccessModal`.

Or even better, change `GeneralSuccessModal` to `ResultModal` and control whether it shows success/failure using props.

Reply:
Messeage-delete-specific `ResultModal`? Or a general component?

Reply:
A general component. Like a sibling of `ConfirmModal`.